docs/examples/logs/README.rst

```diff
@@ -15,6 +15,7 @@ Start the Collector locally to see data being exported. Write the following file
       otlp:
         protocols:
           grpc:
+            endpoint: 0.0.0.0:4317
```

Comment:
I know we have the otel-collector-config.yaml, but could you please also update this block with the same content from the yaml file? It seems different and without the traces pipeline the example won't complete without errors

Reply:
Yes, the README and the otel-collector content were different and have been changed. Thanks.